lib/v2/mihoyo/bbs/cache.js

```diff
@@ -0,0 +1,33 @@
+const got = require('@/utils/got');
+
+module.exports = {
+    getUserFullInfo: (ctx, uid) => {
+        const key = 'mihoyo:user-full-info-uid-' + uid;
+        return ctx.cache.tryGet(key, async () => {
+            const query = new URLSearchParams({
+                uid,
+                gids: 2,
+            }).toString();
+            const url = `https://bbs-api.miyoushe.com/user/wapi/getUserFullInfo?${query}`;
+            const response = await got({
+                method: 'get',
+                url,
+                headers: {
+                    Referer: `https://www.miyoushe.com/ys/accountCenter/postList?id=${uid}`,
+                },
+            });
```

Comment:
You can pass the query string into got's `searchParams` option
docs: https://github.com/sindresorhus/got/tree/v11.8.6?tab=readme-ov-file#url
https://github.com/sindresorhus/got/tree/v11.8.6?tab=readme-ov-file#searchparams